packages/authentication/README.md

```diff
@@ -192,7 +208,7 @@ For more info about passport, see [passport.js](http://passportjs.org/).
 
 ## Contributions
 
-- [Guidelines](https://github.com/strongloop/loopback-next/wiki/Contributing#guidelines)
+- [Guidelines](https://github.com/strongloop/loopback-next/blob/master/docs/DEVELOPING.md)
```

Comment:
The link should be https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md now